### PR TITLE
doc: improve the Kubernetes extension docs slightly

### PR DESCRIPTION
**Pull request by Ladicek**

### CODE REVIEW

File: docs/src/main/asciidoc/kubernetes.adoc [hunk 1]
```diff
@@ -142,21 +142,21 @@ By default the Kubernetes resources do not contain readiness and liveness probes
 The values of the generated probes will be determined by the configured health properties: `quarkus.smallrye-health.root-path`, `quarkus.smallrye-health.liveness-path` and `quarkus.smallrye-health.readiness-path`.
 More information about the health extension can be found in the relevant link:microprofile-health[guide].
 
-=== Using the kubernetes client
+=== Using the Kubernetes client
 
-Applications, that are deployed to kubernetes and need to access the API server, will usually make use of the `kubernetes-client` exetension:
+Applications that are deployed to Kubernetes and need to access the API server, will usually make use of the `kubernetes-client` exetension:
```
Comment (gsmet) [inline docs/src/main/asciidoc/kubernetes.adoc]:
```suggestion
Applications that are deployed to Kubernetes and need to access the API server, will usually make use of the `kubernetes-client` extension:
```

**Review by gsmet — CHANGES_REQUESTED** [review]
I spotted another preexisting typo.

**Comment by Ladicek** [comment]
Yay, thanks :-)

**Review by iocanel — APPROVED** [review]
LGTM

**Review by gsmet — APPROVED** [review]

**Comment by gsmet** [comment]
I squashed the 2 commits. Thanks!

**Comment by gsmet** [comment]
I squashed the 2 commits. Thanks!

**Pull request merged**